renovate.json

```diff
@@ -1,6 +1,9 @@
 {
-  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": [
-    "local>laminas/.github:renovate-config"
-  ]
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": [
+        "local>laminas/.github:renovate-config"
+    ],
+    "constraints": {
+        "php": "^7.4"
```

Comment:
I don't *think* so. The reason we did on mezzio was because `composer/composer` was listed in `require-dev` as `^1.5 || ^2.0` which I think was confusing it.

Reply:
I've checked the list of installed packages and `composer/composer` isn't listed, so this should be good to go as is, and if not, I'll submit another PR to fix.